ui/apps/platform/src/Containers/AccessControl/PermissionSets/ResourceDescription.tsx

```diff
@@ -24,7 +24,7 @@ const resourceDescriptions: Record<ResourceName, string> = {
         'Read: View recent compliance runs and their completion status. Write: Trigger compliance runs.',
     Config: 'Read: View options for data retention, security notices, and other related configurations. Write: Modify options for data retention, security notices, and other related configurations.',
     DebugLogs:
-        "Read: View the current logging verbosity level of all components, including Central, Scanner, Sensor, Collector, and Admission controller. Download diagnostic bundle. Note: The diagnostic bundle contains information about all clusters and namespaces regardless of the user's role and access scope. Don't give this permission to users with limited access scope. Write: Modify the logging verbosity level.",
+        "Read: View the current logging verbosity level of all components, including Central, Scanner, Sensor, Collector, and Admission controller. Download diagnostic bundle. Note: The diagnostic bundle contains information about all clusters and namespaces regardless of the user's role and access scope. It also contains information about access control, notifier integrations, and system configuration, regardless of the user's role. Don't give this permission to users with limited access scope. Write: Modify the logging verbosity level.",
```

Comment:
@pedrottimark 
**question:** When you say `access control, notifier integrations, and system configuration` do you mean the information stored about RHACS which is different from `information about all clusters and namespaces` ?

Reply:
In addition to diagnostic bundle, support has needed to ask customers to provide screen grabs from UI to see these configurations. Starting in next release, additional information will be included in diagnostic bundle.

Can you double check that the existing and new sentences make clear that DebugLogs resource permission gives access to information via diagnostic bundle:
* about clusters and namespaces which might not be allowed in UI according to Cluster or Namespace resource permissions or according to access scope of the role
* about these additional "global" resources which might not be allowed in UI according to their respective resource permissions

Because of the above, DebugLogs resource permission is No Access by default for new permission sets.

Reply:
A reason for separate sentence is that access scope is not relevant for access control, notifier integrations, and system configuration.

Reply:
@pedrottimark I suggest the following for the `Read` permission:

**suggestion:** 
View the current logging verbosity level of all components, including Central, Scanner, Sensor, Collector, and Admission controller. 
Download the diagnostic bundle. 
Important: The diagnostic bundle contains sensitive information, not dependent on the user's role and access scope. The diagnostic bundle includes information about all clusters and namespaces, access control, notifier integrations, and system configuration. Do not give this permission to users with limited access scope. 

---

And for the latter part 
> Can you double check that the existing and new sentences ...

I'm not sure what you mean? I don't see those sentences in PR.

Reply:
Super suggested changes. Here are pictures cropped and in conext:

![DebugLogs-2-crop](https://user-images.githubusercontent.com/11862657/168161413-7465191b-ba96-4ff6-917f-2d704cd2f54b.png)

![DebugLogs-2](https://user-images.githubusercontent.com/11862657/168161222-fe63834f-8971-4a18-b78d-52d35fb74452.png)

Sorry I was not clear. Disregard the double check comment.

Reply:
👍🏼  @pedrottimark 

**thought:** I noticed that the screenshot is missing the description for `Write:` and maybe that's why the word `Read` is not in bold.

Reply:
Uh oh, that is a mistake when I pasted into the string. Here is a picture with the **Write** sentence restored.
![DebugLogs-3-crop](https://user-images.githubusercontent.com/11862657/168280857-63eff3c5-f056-435e-99fa-9445aa5f82c0.png)